Properly check if wayland can be used

Dependency wayland-client found: NO (tried pkgconfig and cmake)
Dependency libva-wayland found: NO (tried pkgconfig and cmake)

meson.build:94:2: ERROR:  Unknown variable "wl_scanner".

--- meson.build.orig	2018-12-10 01:56:28 UTC
+++ meson.build
@@ -91,7 +91,7 @@ if get_option('with_wayland') != 'no'
     version : libva_version,
     required : get_option('with_wayland') == 'yes')
 
-  WITH_WAYLAND = wl_scanner.found() and libva_wayland_dep.found()
+  WITH_WAYLAND = wayland_client_dep.found() and libva_wayland_dep.found()
 endif
 
 subdir('src')
